### PR TITLE
Urls scanner 1.0.6

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by imranMnts** [comment]
Hello @imrekel 

Can you review my PR or if I can have any status about that, please.

File: steps/urls-scanner/1.0.6/step.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+title: Mobile apps URLS scanner
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:1]:
```suggestion
title: Mobile apps URL scanner
```

File: steps/urls-scanner/1.0.6/step.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+summary: |
+  Scan source code to extract all http/https urls and execute an analyze of those urls with SSLLabs
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:3]:
```suggestion
  Scan source code to extract all HTTP/HTTPS URLs and execute an analysis of those URLs with SSLLabs
```

File: steps/urls-scanner/1.0.6/step.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+description: |
+  Scan source code to extract all http/https urls and execute an analyze of those urls with SSLLabs
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:5]:
```suggestion
  Scan source code to extract all HTTP/HTTPS URLs and execute an analysis of those URLs with SSLLabs
```

File: steps/urls-scanner/1.0.6/step.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  brew:
+  - name: ssllabs-scan
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- opts:
+    description: |
+      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
+      The list set by default can help you to remove urls that are only used for development perspective.
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:36]:
```suggestion
      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
      The list set by default can help you to remove URLs that are only used from a development perspective.
```

File: steps/urls-scanner/1.0.6/step.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    is_expand: true
+    is_required: false
+    summary: Analyze HTTPS URLs with SSLLabs and generate report for each domain
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:39]:
```suggestion
    summary: Analyze HTTPS URLs with SSLLabs and generate a report for each domain
```

File: steps/urls-scanner/1.0.6/step.yml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    title: Analyze HTTPS URLs with SSLLabs
+    value_options:
+    - "yes"
+    - "no"
+  ssl_labs_scan: "no"
+- black_list: apache.org;cloudflare.com;bintray.com;w3.org;apple-mapkit.com;apple.com;mzstatic.com;mozilla.org;github.io;github.com;youtube.com;withgoogle.com;android.com;google.com;jitpack.io;microsoft.com;firebaseio.com
+  opts:
+    description: |
+      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
+      The list set by default can help you to remove urls that are only used for development perspective.
```
Comment (sylank) [inline steps/urls-scanner/1.0.6/step.yml:49]:
```suggestion
      Define the list of domains like this: google.com;apple.com;flutter.com;bitrise.io (using ; as separator).
      The list set by default can help you to remove URLs that are only used for a development perspective.
```

**Review by sylank — CHANGES_REQUESTED** [review]
@imranMnts I have just suggested some minor typo issues

File: steps/urls-scanner/1.0.6/step.yml (hunk 6 above)
Comment (imranMnts) [reply steps/urls-scanner/1.0.6/step.yml:49]: Thanks a lot, @sylank for the feedback
I applied your changes here #2981

**Comment by imranMnts** [comment]
Moved to #2981

**Pull request closed without merging**